easybuild/easyconfigs/f/FragGeneScanPlus/FragGeneScanPlus-20161116-GCCcore-8.2.0.eb

```diff
@@ -0,0 +1,39 @@
+# #
+# This is a contribution from HPCNow! (http://hpcnow.com)
+# Copyright::   HPCNow!
+# Authors::     Pau Ruiz <pau.ruiz@hpcnow.com>, Arturo Gimeno
+# License::     GPL-v3.0
+# #
+
+easyblock = 'MakeCp'
+
+name = 'FragGeneScanPlus'
+version = '20161116'
+
+homepage = 'https://github.com/hallamlab/FragGeneScanPlus'
+description = """FragGeneScan-Plus for scalable high-throughput short-read open reading frame prediction"""
+
+toolchain = {'name': 'GCCcore', 'version': '8.2.0'}
+
+builddependencies = [('binutils', '2.31.1')]
+
+sources = [{
+    'filename': '%(name)s-%(version)s.tar.gz',
+    'git_config': {
```

Comment:
Using `git_config` triggers a git clone by EasyBuild, which then tars the sources. That leads to non-deterministic checksums. 

You can actually get this same source at https://github.com/hallamlab/FragGeneScanPlus/archive/91b0ab6.tar.gz , which I believe _should_ give you a reproducible commit. See e.g. https://github.com/easybuilders/easybuild-easyconfigs/blob/4432f2762c94ee556f58c7fe8ab28d3f286b466b/easybuild/easyconfigs/p/pfind/pfind-20220613-gompi-2021b.eb#L5

Even more elegant than that example: I think you can specify
```
source_urls = GITHUB_SOURCE
github_account=hallamlab
sources = [local_commit + '.tar.gz']
```
but you'd have to give that a try (see https://docs.easybuild.io/version-specific/easyconfig-templates/#template-constants-that-can-be-used-in-easyconfigs fro the use of templates in EasyConfigs).

Reply:
Yep, better example: https://github.com/easybuilders/easybuild-easyconfigs/blob/4432f2762c94ee556f58c7fe8ab28d3f286b466b/easybuild/easyconfigs/m/MARS/MARS-20191101-GCCcore-8.3.0.eb#L7